dask_saturn/core.py

```diff
@@ -177,14 +203,34 @@ def adapt(self, minimum, maximum):
         if not response.ok:
             raise ValueError(response.reason)
 
-    def close(self):
+    async def _close(self):
+        while self.status == "closing":
+            await asyncio.sleep(1)
```

Comment:
should we exp backoff here?

Reply:
Meh, I don't really care one way or the other. It doesn't have the potential for taking as long as start because there is no potential need to scale up the kubernetes cluster. I guess I can use it for consistency.